app/R/check_battery.R

```diff
@@ -26,8 +26,8 @@ check_battery_daily <- function(daily) {
   data.validator::validate(daily, name = "Daily") |>
     validate_if(btwn(meta_bat_volt_mean, meta_bat_volt_min, meta_bat_volt_max),
                 description = "`meta_bat_volt_*` (min ≤ mean ≤ max)") |> 
-    validate_if(meta_bat_volt_min >= 9.6, "`meta_bat_volt_min` ≥ 9.6") |> 
-    validate_if(meta_bat_volt_max <= 16, "`meta_bat_volt_max` ≤ 16") |> 
+    validate_if(gte(meta_bat_volt_min, 9.6, na_pass = TRUE), "`meta_bat_volt_min` ≥ 9.6") |> 
```

Comment:
It's good for me to see `helpers.R` in action! The example will (hopefully!) be in the back of my mind as my coding adventures continue.

Reply:
Yeah, these were my solution for the fact that you can't add `na.rm = TRUE` to `<=`.  There's maybe a function out there already that does this, but I think this works pretty robustly.